packages/javascript-sdk/rollup.config.js

```diff
@@ -2,5 +2,14 @@
 
 module.exports = (config) => {
   config.output.name = 'forgerock';
+  config.output.globals = {};
+  if (config.output.format === 'umd') {
+    config.output.entryFileNames = '[name].umd.js';
+    config.output.chunkFileNames = '[name].umd.js';
+    return config;
+  }
+  config.output.entryFileNames = '[name].esm.js';
+  config.output.chunkFileNames = '[name].esm.js';
```

Comment:
If we are changing the filenames, would this be considered a breaking change? What about our docs?

Reply:
<img width="1372" alt="Screen Shot 2022-08-30 at 4 34 06 PM" src="https://user-images.githubusercontent.com/18267769/187555354-807a4a2b-3843-4320-86ce-4b351bdd2f8c.png">

https://unpkg.com/browse/@forgerock/javascript-sdk@3.3.1/bundles/index.umd.js
https://unpkg.com/browse/@forgerock/javascript-sdk@3.3.1/bundles/index.esm.js

should be the same.